css/style.css

```
@@ -16,4 +17,175 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.container {
+    margin: 0 auto;
+}
+
+h2, h3 {
+    font-weight: 300;
+
+}
+/*Header*/
+
+ header {
+    display: flex;  
+}
+
+.logo {
+    width: 2em;
+    padding-left: 2em;
+}
+
+nav {
+    display: flex;
+    padding: 1em 0;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+}
+
+li {
+    padding: 0 1em;
+    cursor: pointer;
+}
+
+li:first-child {
+    font-weight: bold;
+}
+
+li:hover {
+    color:#F15B2A;
+}
+
+/*Jumbotron*/
+
+h1 {
+    font-weight: 300;
+    font-size: 2.5em;
+    opacity: 0.9;
+}
+
+ .jumbotron {
+     display: flex;
+     flex-direction: column;
+     align-items: center;
+     justify-content: space-around;
```

```suggestion
     justify-content: center;
```
It seems we want to centre the content, so we can use `center` which is exactly what we want